### ISSUE TITLE
Enable/disable arbitrary settings (Shizuku)

### ISSUE THREAD

**Issue by githubsterer** [keymapperorg/KeyMapper#1634]
**Developer TODO (don't remove)**
- [ ] update documentation
Is there any way to create such an action that would toggle the one handed mode on and off? It is vitally needed to use accessibility options since they cannot both be on at the same time. 

I'm not talking about turning it on and off in use, but enabling and disabling the setting itself.

**Comment by jambl3r** [comment]
> It is vitally needed to use accessibility options since they cannot both be on at the same time.

What do you mean by this?

> I'm not talking about turning it on and off in use, but enabling and disabling the setting itself.

What is the use of this since you shouldn't need to disable it if you want to use it?

**Comment by jambl3r** [comment]
Why would you want to enable the setting more than once? Surely once it's enabled, it can stay enabled forever?

**Comment by githubsterer** [comment]
Because when enabled the magnifier gets disabled. So leaving it enabled means never using the magnifier even when you're not using one handed mode actively. 
 
 
  On Wed, Apr 9, 2025 at 2:51 PM, Jack ***@***.***> wrote:    
Why would you want to enable the setting more than once? Surely once it's enabled, it can stay enabled forever?

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***>
jambl3r left a comment (keymapperorg/KeyMapper#1634)
Why would you want to enable the setting more than once? Surely once it's enabled, it can stay enabled forever?

**Comment by jambl3r** [comment]
If so that sounds like a Samsung problem. I have no trouble using the Android magnifier and one handed mode at the same time

![Screenshot_20250409-223404.png](https://github.com/user-attachments/assets/6d18a8b5-be3b-4988-a0e7-cb0b8687f265)

**Comment by jambl3r** [comment]
May be possible to do this enabling/disabling of random settings with Shizuku, so I'll repurpose this issue for that

**Comment by githubsterer** [comment]
Could be. When you enable OHM it warns that all Accessibility features will be turned off.

    On Wednesday, April 9, 2025 at 05:35:35 PM EDT, Jack Ambler ***@***.***> wrote:  
 
  
If so that sounds like a Samsung problem. I have no trouble using the Android magnifier and one handed mode at the same time

Screenshot_20250409-223404.png (view on web)

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***>
jambl3r left a comment (keymapperorg/KeyMapper#1634)
If so that sounds like a Samsung problem. I have no trouble using the Android magnifier and one handed mode at the same time

Screenshot_20250409-223404.png (view on web)

**Comment by sds100** [comment]
duplicate of #1871
